src/main/java/org/fiware/iam/ServiceMapper.java

```diff
@@ -86,17 +106,25 @@ default CredentialVO map(Credential credential) {
 	 * Map a list of string-entries, encoding TrustedParticipants endpoints to a list of {@link EndpointEntry} with
```

Comment:
Vlad would say "wrong documentation is worse than missing documentation" :laughing:

Reply:
![img](https://github.com/user-attachments/assets/b7c0c6db-f684-402b-81dd-b0b584f78c1e)